WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrderViewModel.swift

```diff
@@ -406,6 +416,16 @@ extension NewOrderViewModel {
                                                             didSelectSave: saveFeeLineClosure)
         }
     }
+
+    /// Representation of order notes data display properties
+    ///
+    struct CustomerNoteDataViewModel {
+        let customerNotes: String
+
+        init(customerNotes: String) {
+            self.customerNotes = customerNotes
+        }
```

Comment:
I think you don't need this initializer since it is automatically synthesized by swift for internal types!

Reply:
Great to know that swift can do that! Nice one! Thanks!